config/prod.json

```diff
@@ -11,7 +10,7 @@
   "firecloudBucketRoot": "https://storage.googleapis.com/firecloud-alerts",
   "firecloudUrlRoot": "https://portal.firecloud.org",
   "externalCreds": {
-    "providers": [],
+    "providers": ["fence", "dcf-fence", "anvil", "kids-first"],
```

Comment:
Should we add "github" here? Or is there more work to be done to only enable it for certain users?

Reply:
I don't think it's ready for prod yet, and we're waiting on Workflows for the go-ahead.